smoke_tests.sh

```diff
@@ -1,9 +1,4 @@
 #!/usr/bin/env bash
-. /opt/smoke.sh/smoke.sh
+set -ex
 
-bin/console --version --env=$ENVIRONMENT_NAME
```

Comment:
the `curl` command is equivalent to the previous smoke test, but doesn't check the body (it's not a massively useful smoke testing framework)
the console command should still be there to check the cli bootstrap works, if `ENVIRONMENT_NAME` is a problem for using the script locally, you can add a default with `"${ENVIRONMENT_NAME:-dev}`